setup.cfg

```diff
@@ -0,0 +1,5 @@
+[flake8]
+inline-quotes = "
+multiline-quotes = "
+ignore = C815, C816, I202
```

Comment:
I think these may now be fixed with the latest flake8?

Reply:
I just used our most common linting config here, but this repo is so tiny that I doubt any of these will actually raise any issues.